src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseSteg.kt

```diff
@@ -61,18 +60,18 @@ class TilsynBarnBeregnYtelseSteg(
         beregningsresultat: BeregningsresultatTilsynBarnDto,
     ) {
         val andelerTilkjentYtelse = beregningsresultat.perioder.flatMap {
-            it.grunnlag.stønadsperioderGrunnlag.map { stønadsperiodeMedAktivitet ->
+            it.beløpsperioder.map { beløpsperiode ->
                 AndelTilkjentYtelse(
-                    // TODO hvordan burde vi egentligen gjøre med decimaler?
-                    beløp = it.dagsats.setScale(0, RoundingMode.HALF_UP).toInt(),
-                    fom = stønadsperiodeMedAktivitet.stønadsperiode.fom,
-                    tom = stønadsperiodeMedAktivitet.stønadsperiode.tom,
-                    satstype = Satstype.DAG, // TODO
-                    type = TypeAndel.TILSYN_BARN_AAP, // TODO
+                    beløp = beløpsperiode.beløp,
+                    fom = beløpsperiode.dato,
+                    tom = beløpsperiode.dato,
+                    satstype = Satstype.ENGANGSBELØP,
```

Comment:
Skal fortsatt være DAG
```suggestion
                    satstype = Satstype.DAG,
```